packages/spatial/src/input/properties/InputComponentNodeEditor.tsx

```diff
@@ -0,0 +1,141 @@
+/*
+CPAL-1.0 License
+
+The contents of this file are subject to the Common Public Attribution License
+Version 1.0. (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+https://github.com/EtherealEngine/etherealengine/blob/dev/LICENSE.
+The License is based on the Mozilla Public License Version 1.1, but Sections 14
+and 15 have been added to cover use of software over a computer network and 
+provide for limited attribution for the Original Developer. In addition, 
+Exhibit A has been modified to be consistent with Exhibit B.
+
+Software distributed under the License is distributed on an "AS IS" basis,
+WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the
+specific language governing rights and limitations under the License.
+
+The Original Code is Ethereal Engine.
+
+The Original Developer is the Initial Developer. The Initial Developer of the
+Original Code is the Ethereal Engine team.
+
+All portions of the code written by the Ethereal Engine team are Copyright © 2021-2023 
+Ethereal Engine. All Rights Reserved.
+*/
+
+import { useQuery, UUIDComponent } from '@etherealengine/ecs'
+import { getComponent, useComponent } from '@etherealengine/ecs/src/ComponentFunctions'
+import { PropertiesPanelButton } from '@etherealengine/editor/src/components/inputs/Button'
+import InputGroup from '@etherealengine/editor/src/components/inputs/InputGroup'
+import NumericInput from '@etherealengine/editor/src/components/inputs/NumericInput'
+import SelectInput from '@etherealengine/editor/src/components/inputs/SelectInput'
+import NodeEditor from '@etherealengine/editor/src/components/properties/NodeEditor'
+import {
+  commitProperties,
+  commitProperty,
+  EditorComponentType,
+  updateProperty
+} from '@etherealengine/editor/src/components/properties/Util'
+import { EditorControlFunctions } from '@etherealengine/editor/src/functions/EditorControlFunctions'
+import { SourceComponent } from '@etherealengine/engine/src/scene/components/SourceComponent'
+import { NameComponent } from '@etherealengine/spatial/src/common/NameComponent'
+import PanToolIcon from '@mui/icons-material/PanTool'
+import React, { useEffect } from 'react'
+import { useTranslation } from 'react-i18next'
+import { InputComponent } from '../components/InputComponent'
+
+type OptionsType = Array<{
+  label: string
+  value: string
+}>
+
+export const InputComponentNodeEditor: EditorComponentType = (props) => {
```

Comment:
this should not be in the spatial package, it should be in the editor package. might crash builder or servers.

Reply:
moved, thanks for the reminder!